ariac_gazebo/ariac_gazebo/environment_startup.py

```diff
@@ -1314,6 +1318,12 @@ def parse_conveyor_config(self):
 
                 self.conveyor_parts_to_spawn.append(PartSpawnParams(
                     part_name, part.type, part.color, xyz=xyz, rpy=rpy))
+ 
+                self.conveyor_parts_to_spawn_cycle = cycle(self.conveyor_parts_to_spawn)
+
+                shuffle(self.conveyor_parts_to_spawn)
+                
+                self.conveyor_parts_to_spawn_random_cycle = cycle(self.conveyor_parts_to_spawn)
```

Comment:
why do these need to be two separate cycles? Couldn't you just shuffle the list before you create the cycle if the spawn order is random and not shuffle it if it is sequential?